CMakeLists.txt

```diff
@@ -1054,7 +1054,7 @@ endif()
 if(HAVE_UNISTD_H)
   SET(ZCONF_UNISTD_LINE "#if 1    /* was set to #if 1 by configure/cmake/etc */")
 else()
-  SET(ZCONF_UNISTD_LINE "#ifdef HAVE_UNISTD_H    /* may be set to #if 1 by configure/cmake/etc */")
+  SET(ZCONF_UNISTD_LINE "#if ~ HAVE_UNISTD_H + 1    /* may be set to #if 1 by configure/cmake/etc */")
```

Comment:
Can you explain this logic a bit more?

Reply:
If something is not defined, it equals either 0 or empty string...

Reply:
This has the exact same semantics as here [1]; it is only written slightly shorter.

[1]: https://github.com/zlib-ng/zlib-ng/blob/c91806258805d243080a934ebd169333d4f8ba46/zconf.h.in#L152

Reply:
I don't think shorter is better in this case... It is just confusing...

Reply:
IMO `-_LARGEFILE64_SOURCE - -1 == 1` is just as unreadable.

Reply:
Possibly... But fixing both would change scope of this pull request...